frontend/package.json

```diff
@@ -24,9 +24,9 @@
     "query-string": "7.1.1",
     "react": "18.2.0",
     "react-avatar-editor": "13.0.0",
+    "react-content-loader": "6.2.0",
```

Comment:
why you cant use "react-loading-skeleton" instead this dependecy?

Reply:
i use this dependency in previos task to implement skeleton to video card from search page in my opinion it is easier to use